repos/fdbt-site/src/pages/managePurchaseMethod.tsx

```diff
@@ -269,11 +289,14 @@ const ManagePurchaseMethod = ({ inputs, csrfToken, errors, editMode }: ManagePur
 export const getServerSideProps = async (
     ctx: NextPageContextWithSession,
 ): Promise<{ props: ManagePurchaseMethodsProps }> => {
-    return await getGlobalSettingsManageProps<FromDb<SalesOfferPackage>>(
+    const globalSettingsProps = await getGlobalSettingsManageProps<FromDb<SalesOfferPackage>>(
         ctx,
         getSalesOfferPackageByIdAndNoc,
         getSessionAttribute(ctx.req, GS_PURCHASE_METHOD_ATTRIBUTE),
     );
+
+    const isCapped = !!ctx.query.capped;
```

Comment:
this will return true if the URL is ?capped=hello. We should be checking the value of capped to === 'true'